module_report/check_apis.py

```diff
@@ -0,0 +1,12 @@
+import os
+import sys
+import subprocess
+
+import qtpy
+
+
+for api in qtpy.API_NAMES:
```

Comment:
```suggestion
def check_apis(report_name)
    for api in qtpy.API_NAMES:
```

Sorry, I screwed this up in my previous suggestion by accidentally deleting the function signature.

Reply:
I already fixed it locally.